### PR TITLE
Port Services page and plots from moment to Date/Intl

### PR DESCRIPTION
**Pull request by martinpitt**
Plots on main:
main-no-year
main-with-year

Plots on this PR:
pr-no-year
pr-with-year

Timers on main:
main-timers

Timers on this PR:
pr-timers

### CODE REVIEW

**Comment by martinpitt** [comment]
[This failure](https://logs.cockpit-project.org/logs/pull-16028-20210701-080219-01d7f786-rhel-9-0/log.html#83) is the same as we already found in https://github.com/cockpit-project/bots/pull/2144 , so it's not as rare as we hoped. This needs to be handled. @KKoukiou FYI

**Comment by KKoukiou** [comment]
> This fail

Ack, I will handle this today.

**Comment by martinpitt** [comment]
I sent https://github.com/cockpit-project/bots/pull/2163 for the SELinux nft issue.

**Comment by martinpitt** [comment]
Re-running tests to pick up new naughty.

File: pkg/systemd/services/services.jsx [hunk 1]
```diff
@@ -70,8 +70,8 @@ export function updateTime() {
                 clock_monotonic_now = parseInt(uptime * 1000000, 10);
             }, ex => console.log(ex.toString()));
     cockpit.spawn(["date", "+%s"])
-            .then(function(time) {
-                clock_realtime_now = moment.unix(parseInt(time));
+            .then(time => {
+                clock_realtime_now = parseInt(time, 10) * 1000;
```
Comment (KKoukiou) [inline pkg/systemd/services/services.jsx:74]: explanation: moment.unix is moment(timestamp * 1000)

I had to check this one.

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**